wright/src/grammar/parsers/whitespace.rs

```diff
@@ -1,22 +1,35 @@
 use crate::grammar::model::Fragment;
-use nom::character::complete::{char as ch, multispace0, not_line_ending};
+use nom::branch::alt;
+use nom::bytes::complete::{tag, take_until};
+use nom::character::complete::{char, multispace0, not_line_ending};
```

Comment:
In general, I am trying to import nom's char parser as ch (to avoid conflicting with std::char or the char intrinsic)